imperial_coldfront_plugin/models.py

```diff
@@ -29,3 +29,24 @@ class GroupMembership(models.Model):
         on_delete=models.CASCADE,
         related_name="group_memberships_set",
     )
+
+
+class UserID(models.Model):
```

Comment:
I would go for `UnixUID` here as it's less ambiguous.

Reply:
Done!